graylog2-server/src/test/java/org/graylog2/database/MongoCollectionsTest.java

```diff
@@ -71,11 +69,10 @@ record Secret(@JsonProperty("encrypted_value") EncryptedValue encryptedValue) {}
     record IgnoreTest(@JsonProperty("ignore_me_not") String ignoreMeNot,
                       @MongoIgnore @JsonProperty("ignore_me") String ignoreMe) {}
 
-    record IdGenerationTest(@Nullable @JsonProperty("id") @Id @org.mongojack.ObjectId String id) {}
+    record IdGenerationTest(String id) implements MongoEntity {}
 
     @JsonIgnoreProperties(ignoreUnknown = true)
-    record TimestampTest(@Nullable @JsonProperty("id") @Id @org.mongojack.ObjectId String id,
-                         @JsonProperty("timestamp") DateTime timestamp) {}
+    record TimestampTest(String id, @JsonProperty("timestamp") DateTime timestamp) implements MongoEntity {}
```

Comment:
Same as above. Please check if annotation inheritance works for records. Thanks!